packages/portal-proto/src/components/Modals/SetModals/SaveSelectionModal.tsx

```diff
@@ -100,17 +100,22 @@ const SaveSelectionAsSetModal: React.FC<SaveSelectionAsSetModalProps> = ({
     >
       <div className="p-4">
         <NumberInput
+          data-testid="textbox-number-of-top-items"
           required
           label="Save top:"
           min={1}
           max={max}
           {...form.getInputProps("top")}
         />
-        <p className="text-sm pb-2 pt-1">
+        <p
+          data-testid="text-up-to-top-items-can-be-saved"
+          className="text-sm pb-2 pt-1"
```

Comment:
@bpierce01 are these your changes? to the className?

Reply:
The classname didn't change just the formatting bc of pre-commit

Reply:
Sounds good! Thank you for the clarification!